index.js

```diff
@@ -293,7 +286,7 @@ app.post('/api/xethru', async (req, res) => {
 
 // Handler for income SmartThings POST requests
 app.post('/api/doorTest', async(req, res) => {
```

Comment:
@mariocimet are we currently using this endpoint for tests, or does this exist as a stub for the IM21 to be added later?

Reply:
its for tests, although I don't imagine the im21 endpoint will be much different